src/util.cpp

```diff
@@ -408,3 +409,135 @@ void indent(std::ostream& strm, unsigned level) {
 }
 
 }}
+
+#ifndef HAVE_EPICSPARSE
+PVXS_API int epicsParseInt8(const char* s, epicsInt8* val, int base, char**)
+{
+    try {
+        size_t idx;
+        epicsInt8 v = (std::stoi(s, &idx, base) & 0xff);
+        if (idx == std::string(s).length()) {
```

Comment:
Creating temporary `std::string`s like this is not so efficient.  `strlen()` is entirely appropriate.  Or if you want to be super efficient, test if `s[idx]=='\0'`.  As I [read it](https://en.cppreference.com/w/cpp/string/basic_string/stoul) `idx` will always point to a valid character, which includes the nil.